jujugui/static/gui/src/app/components/usso-login-link/usso-login-link.js

```diff
@@ -49,26 +49,43 @@ YUI.add('usso-login-link', function() {
     },
 
     /**
+       Returns a notification about popups
+     */
+    _renderNotification: function() {
+      return (
+        <div className="usso-login__notification">
+          When requested please allow popups from {window.location.origin} to login
```

Comment:
We should probably give a bit of a hint as to where they should look - the address bar for the icon.